docs/changelog.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+* **18 Feb 2019:**
+  * Added a `build.sh` script to each worker so that it is possible to build a single worker with the script.
```

Comment:
Does "each worker" mean "each worker configuration file", or does it refer to something else?
```suggestion
  * Added a `build.sh` script to each worker so that it is possible to build a single worker instance with the script.
```

Reply:
Ah, should this say "build-project.sh" instead of "build.sh"? And should it mention the folder (`SpatialOS/scripts/`) like the next few bullet points do? (Or is this something different?)